### DISCUSSION TITLE
Stellarium: Space probes and Crosshairs

### DISCUSSION THREAD

**Discussion by StellariumUser** [Stellarium/stellarium#870] [Q&A] [answered]
- [ ] (1.) How i can add space probes like New Horizons, Voyger, Pioneer to Stellarium ?
- [x] (2.)  How i can generate the crosshair as an result of a search / input in the Right asencion - Declination values field ?

Greetings
Dieter

**Comment by alex-w** [comment]
1. No. See https://github.com/Stellarium/stellarium/issues/259
2. A quick workaround at the moment - using CCD mode in Oculars plugin. Are you really want to get a crosshair of center of the screen?

**Comment by alex-w** [comment]
Second point has been implemented since version 0.20.1.17888

**Comment by gzotti** [comment, marked as answer]
And first is a duplicate of #259. (And not related to the Satellite plugin!)
